### ISSUE TITLE
Suggestion: add an option to mute and/or block all bots in public timelines IMMEDIATELY

### ISSUE THREAD

**Issue by KNTRO** [mastodon/mastodon#12322]
### Pitch

Bots and public timelines

### Motivation

I'm pretty sure I'm not alone on this.

How many of you people are tired about seeing bunches of useless bots in your public timelines, tooting all day long? I have to manually mute all bots I have found, since I don't care at all about them. I want to read **toots from people** who actually write them, and not about bots copying and pasting blog entries' title and URL. It's even worse in the case of bots that only toot images. Oh, BTW, did I mention bots tooting p0rn pics? Beautiful, aren't they? 😒

So, **I'm asking for an option to automatically mute all bots from all my public timelines (including notifications)** when I'm logged in. That's it. I want clearer timelines, with **human-generated feeds**. I think this is especially important for people coming from social networks where bots are not THAT normal, and they probably don't care —or even are kind of "scared" of— them.

Trust me, have to mute bots all day is far away from being such a funny thing.

**Comment by trwnh** [comment]
Duplicate of #8293

**Comment by Cassolotl** [comment]
There is a pull request for this that was opened in July 2019, #11272, which was approved 12 days ago and is just waiting on being merged by Gargron.

**Comment by KNTRO** [comment]
@Cassolotl Thanks for this information!
